pgml-extension/src/bindings/transformers.py

```diff
@@ -82,6 +83,27 @@ def ensure_device(kwargs):
             kwargs["device"] = "cpu"
 
 
+class StandardPipeline(object):
+    def __init__(self, model_name, **task):
+        task.pop("model")
+        self.task = task.pop("task")
+        task.pop("device")
+        self.model = AutoModelForCausalLM.from_pretrained(model_name, **task)
+        self.tokenizer = AutoTokenizer.from_pretrained(model_name)
+
+    def __call__(self, inputs, **kwargs):
+        outputs = []
+        pipe = pipeline(
```

Comment:
We don't want to incur the cost of instantiating a new pipeline on every invocation. This is something that should happen during __init__.

Reply:
Don't disagree. I didn't look into if pipeline was an expensive instantiation or really just a wrapper. I put it in the call rather than init to allow different parameters (temp, top_k, etc) per call without having to load a whole new model into ram and cache it.

Reply:
Yup, looks heavy enough to not do that: https://github.com/huggingface/transformers/blob/main/src/transformers/pipelines/base.py#L735